docs/modules/ROOT/pages/guides/v4-migration/authorization.adoc

```diff
@@ -0,0 +1,272 @@
+
+== Instantiation
+
+Whilst the `@auth` directive required the installation of an additional plugin package, the functionality for the new directives is built directly into the library. You should uninstall the previous plugin:
+
+[source, bash, indent=0]
+----
+npm uninstall @neo4j/graphql-plugin-auth
+----
+
+Then, given an example of instantiation using a basic secret with the plugin:
+
+[source, typescript, indent=0]
+----
+new Neo4jGraphQL({
+    typeDefs,
+    plugins: {
+        auth: new Neo4jGraphQLAuthJWTPlugin({
+            secret: "secret",
+        }),
+    }
+})
+----
+
+You can delete the import of `Neo4jGraphQLAuthJWTPlugin` and change the instantiation to:
+
+[source, typescript, indent=0]
+----
+new Neo4jGraphQL({
+    typeDefs,
+    features: {
+        authorization: {
+            key: "secret",
+        }
+    }
+})
+----
+
+== Server
+
+Previously, you could pass in the entire request object and the library would fish the `Authorization` header out:
+
+[source, typescript, indent=0]
+----
+const server = new ApolloServer({
+    schema, // schema from Neo4jGraphQL.getSchema()
+});
+
+const { url } = await startStandaloneServer(server, {
+    listen: { port: 4000 },
+    context: async ({ req }) => ({ req }),
+});
+----
+
+With the new implementation, the library expects the `Authorization` header to be extracted in the format of a bearer token in the `token` field of the context:
+
+[source, typescript, indent=0]
+----
+const server = new ApolloServer({
+    schema, // schema from Neo4jGraphQL.getSchema()
+});
+
+const { url } = await startStandaloneServer(server, {
+    listen: { port: 4000 },
+    context: async ({ req }) => ({
+        token: req.headers.Authorization,
+    }),
+});
+----
+
+This is to acknowledge the fact that there are a variety of servers which don't have a `req` object (such as serverless functions, which use `event`). This is a uniform experience, no matter what server you use.
+
+=== `rolesPath`
+
+`rolesPath` was used to configure a custom path for the "roles" claim in the JWT structure. This configuration has now been moved into the type definitions themselves. So given a previous instantiation:
+
+[source, typescript, indent=0]
+----
+new Neo4jGraphQL({
+    typeDefs,
+    plugins: {
+        auth: new Neo4jGraphQLAuthJWTPlugin({
+            secret: "secret",
+            rolesPath: "some.nested.path",
+        }),
+    }
+})
+----
+
+This will now need to instead be configured in the type definitions as:
+
+[source, graphql, indent=0]
+----
+type JWT @jwt {
+    roles: [String!]! @jwtClaim(path: "some.nested.path")
+}
+----
+
+The type name itself can be anything, as long as it is decorated by `@jwt`.
+
+Whilst this is more boilerplate, the strongly typed nature of the definition means there is significantly more powerful filtering options in version 4.0.0.
```

Comment:
Could we use a simpler word than boilerplate? I'm not sure how this is common among readers. Maybe "standardized"?